api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/ChildDateOfBirthCalculatorTest.java

```diff
@@ -278,10 +283,49 @@ void shouldReturnChildrenTurningOneAndChildTurningFourInPaymentCycle() {
         assertThat(summary).isEqualTo(expectedSummary);
     }
 
+    @ParameterizedTest(name = "Children dobs={0}")
+    @MethodSource("provideArgumentsForChildrenUnderFour")
+    void shouldReturnChildrenAtStartOrPaymentCycle(List<LocalDate> childrenDobs) {
+        //Given a children under 4 in the PaymentCycle
+        PaymentCycle paymentCycle = aPaymentCycleWithChildrenDobs(childrenDobs);
+        //When
+        boolean hasChildren = childDateOfBirthCalculator.hadChildrenUnder4AtStartOfPaymentCycle(paymentCycle);
+        //Then
+        assertThat(hasChildren).isTrue();
+    }
+
+    @ParameterizedTest(name = "Children dobs={0}")
+    @MethodSource("provideArgumentsForChildrenFourAndOver")
+    void shouldReturnNoChildrenAtStartOrPaymentCycle(List<LocalDate> childrenDobs) {
+        //Given a children 4 or over 4 in the PaymentCycle
+        PaymentCycle paymentCycle = aPaymentCycleWithChildrenDobs(childrenDobs);
+        //When
+        boolean hasChildren = childDateOfBirthCalculator.hadChildrenUnder4AtStartOfPaymentCycle(paymentCycle);
+        //Then
+        assertThat(hasChildren).isFalse();
+    }
+
+    private static Stream<Arguments> provideArgumentsForChildrenFourAndOver() {
+        return Stream.of(
```

Comment:
It's not worth changing now, but you could use periods and use a CSV source instead of a method source.